pulp_ansible/app/migrations/0037_gitremote.py

```diff
@@ -17,6 +17,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('remote_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, related_name='ansible_gitremote', serialize=False, to='core.remote')),
                 ('metadata_only', models.BooleanField(default=False)),
+                ('git_ref', models.TextField()),
```

Comment:
Regenerating a migration in a PR sounds like a bad idea.

Reply:
This migration has not been released. So I decided to just update it.

Reply:
This migration has not been released yet.

Reply:
I think, it's still dangerous.